examples/dreamcast/pvr/normalmap/main.cc

```diff
@@ -0,0 +1,307 @@
+/*   
+   bump.elf - an example showing how to use bumpmapping features
+   Copyright (c) 2005 Fredrik Ehnbom
+
+   This software is provided 'as-is', without any express or implied 
+   warranty. In no event will the authors be held liable for any 
+   damages arising from the use of this software.
+
+   Permission is granted to anyone to use this software for any 
+   purpose, including commercial applications, and to alter it and 
+   redistribute it freely, subject to the following restrictions:
+
+   1. The origin of this software must not be misrepresented; you 
+      must not claim that you wrote the original software. If you use
+      this software in a product, an acknowledgment in the product 
+      documentation would be appreciated but is not required.
+
+   2. Altered source versions must be plainly marked as such, and 
+      must not be misrepresented as being the original software.
+
+   3. This notice may not be removed or altered from any source 
+      distribution.
+ */
+
+/*
```

Comment:
You sure you don't want to put your name or anything on this file as an AUTHOR? Maybe even on this comment block to preserve the original one? Cuz you did do a lot that wasn't here...

Reply:
The only work I did on this file was add the comment below that block about Lobotomy modifying the original example it is based on.